components/Navbar.vue

```diff
@@ -59,15 +59,90 @@
       <MobileExpandableSection v-if="isMobile" :title="$t('explore')">
         <NavbarExploreOptions />
       </MobileExpandableSection>
+
+      <MobileExpandableSection
+        v-if="isMobile"
+        :no-padding="true"
+        :title="$t('create')">
+        <b-navbar-item
+          data-cy="classic"
+          :to="`/${urlPrefix}/create`"
+          tag="nuxt-link">
+          {{ $t('classic') }}
+        </b-navbar-item>
+        <b-navbar-item
+          data-cy="simple"
+          :to="`/${urlPrefix}/mint`"
+          tag="nuxt-link">
+          {{ $t('simple') }}
+        </b-navbar-item>
+        <b-navbar-item
+          data-cy="creative"
+          :to="`/${urlPrefix}/creative`"
+          tag="nuxt-link">
+          {{ $t('creative') }}
+        </b-navbar-item>
+      </MobileExpandableSection>
+
       <CreateDropdown
-        v-show="isCreateVisible"
+        v-show="isCreateVisible && !isMobile"
         class="navbar-create custom-navbar-item"
         data-cy="create"
         :chain="chain" />
       <StatsDropdown
+        v-if="!isMobile"
         class="navbar-stats custom-navbar-item"
         data-cy="stats"
         :chain="chain" />
+
```

Comment:
It has the same problem as mentioned above. Could we try to put them into the original component and then, manage to reuse more code.

Reply:
done